src/main/scala/drtlib/SDate.scala

```diff
@@ -0,0 +1,83 @@
+package drtlib
+
+import SDate.{implicits, jodaSDateToIsoString}
+import org.joda.time.{DateTime, DateTimeZone}
+import org.joda.time.format.ISODateTimeFormat
+import org.slf4j.{Logger, LoggerFactory}
+
+import scala.util.Try
+
+case class SDate(dateTime: DateTime) {
```

Comment:
we should have a proper discussion about handling these shared types, but this is cool for now.